cmd/api/main.go

```diff
@@ -1,28 +1,28 @@
 package main
 
 import (
-	"fmt"
 	"net/http"
 
 	"github.com/go-chi/chi/v5"
 	"github.com/joho/godotenv"
-	log "github.com/sirupsen/logrus"
+	"github.com/rs/zerolog/log"
 	"github.com/thirdweb-dev/indexer/internal/handlers"
+	customLogger "github.com/thirdweb-dev/indexer/internal/log"
 )
 
 func main() {
 	err := godotenv.Load()
 	if err != nil {
-		log.Errorf("error loading .env file: %w", err)
+		log.Error().Err(err).Msg("error loading .env file")
 	}
+	customLogger.InitLogger()
```

Comment:
There is no hoisting in go?

Reply:
need to load the env vars before initializing the logger, otherwise it can't be configured